.github/workflows/reusable-release.yml

```diff
@@ -416,9 +411,10 @@ jobs:
           cache: sbt # the issue described in https://github.com/actions/setup-java/pull/564 doesn't affect this step (no version.sbt)
       - name: Release
         env:
-          SONATYPE_USERNAME: ${{ inputs.SONATYPE_USERNAME }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_TOKEN: ${{ secrets.SONATYPE_TOKEN }}
         run: |
+          export SONATYPE_USERNAME="${SONATYPE_TOKEN%%:*}" # See https://github.com/xerial/sbt-sonatype/pull/62
+          export SONATYPE_PASSWORD="${SONATYPE_TOKEN#*:}"
```

Comment:
These two lines will no longer be necessary if https://github.com/xerial/sbt-sonatype/pull/464 is merged & released. Note that `export` is crucial here - without it, `sbt "sonatypeBundleRelease"` doesn't seem to pick up `SONATYPE_USERNAME` & `SONATYPE_PASSWORD`.